build/xwalk.gni

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file contains Crosswalk-feature-related build flags. Users should
+# import it into their build folder by run:
+#   echo 'import("//xwalk/build/xwalk.gni")' > out/your_folder/args.gn'
+# Then run:
+#   gn gen out/your_folder
+
+# Set it false when building content_shell.
+declare_args() {
```

Comment:
I understand we should try to not touch the src/BUILD.gn.  Actually there is only one flag is added in BUILD.gn which is very easy to rebase. On another hand, To some extent, Crosswalk and Chrome are quite similar apps both based on content_layer, I'm not sure if there are more underlying GN target conflicts. Fix those conflicts will also bring new rebase burden.
I'm not very sure about this part, please help to give some suggestions,  Thanks very much.